src/ott/geometry/costs.py

```diff
@@ -279,6 +272,8 @@ class RegTICost(TICost, abc.ABC):
   where :func:`reg` is the regularization function.
   """
 
+  gamma = 0
```

Comment:
Same here, to be as general as possible, the `gamma` isn't needed. Unfortunately, we can't refactor easily to always require gamma (in a clear way), i.e.,
`0.5 * jnp.linalg.norm(z, ord=2) ** 2 + self.gamma * self.reg(z)`, because some regularizers (STVS) weights by `self.gamma ** 2`.